src/main/java/Duke.java

```diff
@@ -37,7 +37,22 @@ public static void main(String[] args) {
                     tm.list();
                     break;
                 default:
-                    tm.handle(command);
+                    try {
```

Comment:
You don't actually have to define so many Exception types. You can try create a method in DukeException to costumize Duke exception message.

Reply:
Is that considered good practice? I was worried that might mean that a DukeException could have multiple meanings, making it ambiguous.